star_tides/exceptions.py

```diff
@@ -0,0 +1,115 @@
+''' star_tides.exceptions.base_exception
+'''
+
+
+
+# LOG LEVELS
+SEVERITY_DEBUG = 'DEBUG'
+SEVERITY_INFO = 'INFO'
+SEVERITY_WARNING = 'WARNING'
+SEVERITY_ERROR = 'ERROR'
+SEVERITY_CRITICAL = 'CRITICAL'
+
+
+# CLIENT RESPONSE CONSTANTS
```

Comment:
These are likely obsoleted via usage of https://docs.python.org/3/library/http.html#http-status-codes, right?